lib/livebook/session.ex

```diff
@@ -2889,34 +2889,9 @@ defmodule Livebook.Session do
   # attributes that are missing.
   defp normalize_runtime_output(output)
 
-  defp normalize_runtime_output(%{type: :input, attrs: attrs} = output)
-       when attrs.type in [:text, :textarea, :password, :number, :url, :range, :color] and
```

Comment:
We only want blur for these types, so we need to reflect that in the new version :)